app/assets/stylesheets/application.scss

```diff
@@ -635,16 +635,16 @@ span.inactive {
 }
 
 .password-toggle-button {
-	color: $blue;
-	bottom: 10px;
-	font-size: 14px;
-	height: 24px;
-	left: 425px;
-	line-height: 0;
-	padding: 0px;
-	position: absolute;
-	width: 47px;
-	z-index: 10;
+  color: $blue;
+  bottom: 10px;
+  font-size: 14px;
+  height: 20px;
+  right: 10px;
```

Comment:
@PriyangaPKini I tested in my local and have a few minor CSS change suggestions:
1. Make `right: 9px`
2. Remove `width`
3. Set `padding-right: 4px` and `padding-left: 4px`

Reply:
Will do these chnages.